things/elevator/modbus/js/main.js

```diff
@@ -70,76 +63,104 @@ const lightSwitchForms = [{
 
 thingDescription['properties']['lightSwitch']['forms'] = lightSwitchForms
 
+const onTheMovePollingTime = 1000
+
 const onTheMoveForms = [{
-    "href": `${thingUnitID}?address=1&quantity=1`,
+    "href": `?address=1&quantity=1`,
     "op": [
-        "readproperty"
+        "readproperty",
+        "observeproperty"
     ],
     "modbus:entity": "DiscreteInput",
     "modbus:function": "readDiscreteInput",
+    "modbus:pollingTime": onTheMovePollingTime,
     "contentType": "application/octet-stream"
 }]
 
 discreteInputs[0] = 0
+let onTheMoveIsPolled = false
 thingDescription['properties']['onTheMove']['forms'] = onTheMoveForms
 
 const floorNumberForms = [{
-    "href": `${thingUnitID}?address=1&quantity=1`,
-    "op": [
-        "readproperty"
-    ],
+    "href": `?address=1&quantity=1`,
+    "op": "readproperty",
     "modbus:entity": "HoldingRegister",
-    "modbus:function": "readHoldingRegisters",
+    "modbus:function": "readHoldingRegister",
     "contentType": "application/octet-stream"
 }, {
-    "href": `${thingUnitID}?address=1&quantity=1`,
-    "op": [
-        "writeproperty"
-    ],
+    "href": `?address=1&quantity=1`,
+    "op": "writeproperty",
     "modbus:entity": "HoldingRegister",
     "modbus:function": "writeSingleHoldingRegister",
     "contentType": "application/octet-stream"
 }]
 
+const minFloorNumber = 0
```

Comment:
These should be in the TM and TD as well

Reply:
There should be `minimum` and `maximum` fields for the property floorNumber with new commits.